packages/block-library/src/post-content/block.json

```diff
@@ -69,4 +69,4 @@
 	},
 	"style": "wp-block-post-content",
 	"editorStyle": "wp-block-post-content-editor"
-}
\ No newline at end of file
+}
```

Comment:
This change looks unintentional and unrelated to the PR - could you revert it before we ship the PR?

Reply:
Hi @tyxla, I would unable to do so, because when I sync the current branch with trunk and rebase it, and did commit, it was done automatically when run linting using precommit hooks.

Reply:
This is odd! It looks completely unrelated.

I'd be happy to help with it, but I'll need permissions. Do you see this in the sidebar of this PR? You might need to tick the checkbox so we have permissions to push to your branch:

<img width="319" alt="Screenshot 2024-11-08 at 1 05 54" src="https://github.com/user-attachments/assets/14fed715-7570-46b0-a77c-ed13c4378c4d">

Reply:
Hi @tyxla, thanks for the help, I see this box is already checked.

Reply:
Hi @tyxla, I had fix this, by adding `--no-verify` and committed the changes.

Thank You,